app/javascript/styles/_explore.scss

```diff
@@ -243,3 +246,24 @@
     border-bottom: 1px solid #ddd;
   }
 }
+
+.options-label {
+  font-size: 1.2em;
+}
+
+.right-margin-10 {
```

Comment:
Blocker: this too tightly couples selector names to style declarations. It negates the value of selectors by hardcoding the exact style into the name. It's like inline styling, but seems more complex to maintain.  It also switches the usual word order of our selectors.

Let's use less specific selector names, to better convey intent and give more flexibility in the style. E.g. `.margin-extra-left` conveys we something farther left than `.margin-left`, or if we think this warrants a unique styling for the element (this selector is only used once) then `.action-reset-view` would also work.

Regarding word order, the [convention](https://github.com/search?q=repo%3Abroadinstitute%2Fsingle_cell_portal_core%20.margin&type=code) is e.g. `.margin-left` or `.margin-right` or `.margin-extra-left`, rather than `.right-margin-<etc>`.

Reply:
[4e68b70](https://github.com/broadinstitute/single_cell_portal_core/pull/1794/commits/4e68b70238f93feffbfb92dfb44e181f08d5ee89)